services/Directory/data/FilterList.json

```
@@ -17342,4 +17342,21 @@
     "licenseId": 8,
     "name": "phishing-filter (Suricata)"
   }
```

might need to add a comma here
```suggestion
  },
```